packages/babel-plugin-proposal-private-methods/test/fixtures/accessors-modern/basic/exec.js

```diff
@@ -0,0 +1,31 @@
+class Cl {
```

Comment:
nit: I prefer `-to-private-property` over `-modern` since modern assumptions will eventually become common and legacy over time and it may confuse new contributors what does `modern` mean.